### DISCUSSION TITLE
How can I understand which photos belong to one observation? (Group photos into observation)

### DISCUSSION THREAD

**Discussion by laser13** [rcloran/lr-inaturalist-publish#9] [General]
Any visual marks?

**Comment by rcloran** [comment]
The only way I'm aware of is by looking at the metadata panel. Personally I have a custom metadata tagset that shows it, but you should be able to see it if you choose "All plug-in metadata" as your metadata view.

This is definitely something that could be better, but I haven't figured out the best way to do it within the Lightroom constraints. 

I'd be happy to add a tagset with the plug-in if that would help you, or implement/experiment with any other ideas you might have to do this, as it'd certainly be useful.

**Comment by laser13** [comment]
Would it be possible to group observations (in published photos) by date?

**Reply by rcloran** [reply]
Unfortunately publish plugins don't any sort of control like that. All it gets to do is know about the collections that are created or create new collections, and the collections behave (mostly) like normal LR collections... And, as far as I know, there's no way to group photos by date in LR's library view.
